ext/googletest.cmd

```diff
@@ -15,5 +15,5 @@ cd build
 : # on Windows:
 : # https://github.com/google/googletest/blob/main/googletest/README.md#visual-studio-dynamic-vs-static-runtimes
 cmake -G Ninja -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release -DBUILD_GMOCK=OFF -Dgtest_force_shared_crt=ON ..
-cmake --build .
+ninja
```

Comment:
I am thinking about adding an `ext/touch.cmd` file that we can modify to force the cache files to be recreated.

Reply:
Would deleting the existing cache be enough?

https://docs.github.com/en/rest/actions/cache#delete-a-github-actions-cache-for-a-repository-using-a-cache-id

cmake --build . and ninja should be the same thing here, is this for consistency with other files?

Reply:
I didn't know we can delete the existing cache. Thank you for the link to the doc.

I made this change to ext/googletest.cmd simply to force the cache files to be recreated. I didn't see any comment error that I could fix, so I just changed `cmake --build .` to an equivalent command. It wasn't really for consistency with other files.